src/Microsoft.AspNetCore.Routing/DependencyInjection/RoutingServiceCollectionExtensions.cs

```diff
@@ -17,13 +17,11 @@ public static class RoutingServiceCollectionExtensions
     {
         public static IServiceCollection AddRouting(this IServiceCollection services)
```

Comment:
Funny, I just logged a bug on that: https://github.com/aspnet/Routing/issues/284

@kichalla I'll assign to you and it's up to you whether to do in this PR or a new one.